src/WebUI/locales/en.yml

```diff
@@ -502,13 +503,31 @@ character:
         notify:
           warning: A large shield cannot be used on horseback
       upgrade:
-        upgradesTitle: Item Upgrades
+        upgradesTitle: "Item upgrades"
+        tooltip:
+          title: "Upgrade item"
+          description: "You can upgrade your item for a heirloom point"
+        validation:
+          maxRank: "The item already has a maximum rank"
+          loomPoints: "You don't have enough heirloom points"
+          exist: "You already have an improved version of this item"
         confirm:
-          description: 'Are you sure you want to upgrade this item for {loomPoints}? Your item {oldItem} will be changed to {newItem}'
-          value: Upgrade item
+          description: "Are you sure you want to upgrade this item for {loomPoints}? Your item {oldItem} will be changed to {newItem}"
         currentItem: Current item
         notify:
           success: Item successfully upgraded
+      reforge:
+        tooltip:
+          title: "Reforge item"
+          description: "You can reforge your items for a gold"
+        validation:
+          rank: "The item must have a rank greater than {minimumRank}"
+          gold: "You don't have enough gold"
+          exist: "You already have the {existItem} in your inventory. Sell it first"
+        confirm:
+          description: "Are you sure you want to reforge this item for {gold}? You will get {loomPoints}, also item {oldItem} will be changed to {newItem}"
```

Comment:
Would this read as "Are you sure you want to reforge this item for {gold}? You will get **{1}**, also **{+1 hoe}** will be changed to **{hoe}**"?

If yes:
Recommend change to -> "Are you sure you want to reforge this item for {gold}? You will get {loomPoints} loompoint back, in exchange {oldItem} will be changed to {newItem}"

else:
Recommend change to -> "Are you sure you want to reforge this item for {gold}? You will get {loomPoints} back, **in exchange** {oldItem} will be changed to {newItem}"